lib/peep_repository.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'peep'
+
+class PeepRepository
+  def all
+    sql = 'SELECT * FROM peeps ORDER BY timestamp DESC;'
```

Comment:
You generally want to be only doing one thing with each method, so either calling all the peeps or sorting the peeps. Here you are doing both